sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c

```diff
@@ -318,5 +359,25 @@ void stabilization_attitude_set_setpoint_rp_quat_f(struct FloatEulers* indi_rp_c
     QUAT_COPY(q_sp, q_rp_sp);
   }
 
-  QUAT_BFP_OF_REAL(stab_att_sp_quat,q_sp);
+  QUAT_BFP_OF_REAL(stab_att_sp_quat, q_sp);
 }
+
+/**
+ * ABI callback that obtains the acceleration setpoint from telemetry
+ */
+static void accel_sp_cb(uint8_t sender_id, struct FloatVect3 *accel_sp)
+{
+  if (sender_id == 1) {
```

Comment:
I guess these should check for ACCEL_SP_3D_ID and ACCEL_SP_2D_ID

Reply:
actually, I didn't pay enough attention at how the message and ID wax used, it is now done correctly with a flag instead of abusing the sender ID